packages/editor/src/components/post-title/style.scss

```diff
@@ -1,4 +1,16 @@
-.edit-post-text-editor__body .is-raw-text textarea {
-	font-size: inherit;
-	line-height: inherit;
+// Raw Text Variant
+.edit-post-text-editor__body .editor-post-title.is-raw-text {
```

Comment:
What does the `.is-raw-text` class exist for? Even without this class, it seems that we can determine that it is a code editor just by looking at the `.edit-post-text-editor__body` class.

Reply:
Without this class there is no way to determine whether the field is in "rich" or "raw" mode. I prefer not to rely on a cascade of CSS classes in parents that may or not be present in other contexts.

Reply:
I see, I understand 👍